### DISCUSSION TITLE
compleseus completion layer and code browsing.

### DISCUSSION THREAD

**Discussion by kvaneesh** [syl20bnr/spacemacs#15260] [Ideas]
I am wondering whether we can standardize the supported code browsing keybindings.  IIUC we need the below support.

1) Go to definition without/with prompting
2) Go to the declaration without/with prompting
3) find references without/with prompting
4) If the project can use gtags, in dried mode (within any directory inside the project) enable ggtags?. This helps to jump to
project symbol using gtags (with prompting) 

lsp/helm-gtags supports most of these. xref do have support for these but they are not glued
correctly in spacemacs. Wondering whether spacemacs can provide the same.

**Comment by lebensterben** [comment]
you can open an issue for feature request. these are very legit ones we should support.
